facility-manager.html

```diff
@@ -528,6 +541,55 @@
     <div class="line"><br /></div> <!-- end #line -->
 </div> <!-- end #line-block -->
 
+<div class="line-block">
+    <div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h2><a class="anchor" id="lightadmin"></a>Creating administrators with restricted privileges</h2>
+
+<p>This requires using the OMERO.web Admin tool.</p>
+
+<p>This workflow is not implemented in OMERO.insight.</p>
+
+<div class="line-block">
+    <div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h3>OMERO.web</h3>
+
+<ol>
+
+    <li>
+        <p class="first">
+            Click on <span class="bold">Admin</span> in the top
+            toolbar.
+        </p>
+
+        <p class="first">Click on the <span class="bold">Users</span> tab.</p>
+
+        <p>Click on the <span class="bold">Add new User</span> button.</p>
+
+        <p>Fill in the details.</p>
+
+        <p>Click on the radio button <span class="bold">Administrator with restricted privileges</span> in line <span class="bold">Role</span>.</p>
+
+        <p>Hover with mouse over the checkboxes to see short descriptions of the privileges.</p>
+
+        <p>Fill in the table <span class="bold">Choose user privileges</span> by checking the checkboxes.</p>
+
+        <p>Click <span class="bold">Save</span>.</p>
+
+        <div class="line-block">
+            <div class="line"><br /></div> <!-- end #line -->
+        </div> <!-- end #line-block -->
+
+        <img alt="../images/facilityManagerLightAdminWeb.jpg" class="align-center"
+        src="images/facilityManagerLightAdminWeb.jpg" style="width: 100%;" />
+
+    </li>
```

Comment:
Maybe mention somewhere that if you are a Restricted Admin you will only be able to create users with similar of fewer privileges (checkboxes will be disabled).